### PR TITLE
feat(KtUserMenu): make link component in KtUserMenu injectible

### PR DESCRIPTION
**Pull request by HansAuger**
Analogous to what was already done for Navbar

### CODE REVIEW

File: packages/kotti-ui/source/kotti-user-menu/KtUserMenu.vue [hunk 1]
```diff
@@ -77,6 +78,11 @@ export default defineComponent<KottiUserMenu.PropsInternal>({
 			clickawayMenu: () => (isMenuShow.value = false),
 			isMenuShow,
 			isNarrow,
+			parsedSections: computed(() =>
+				props.sections
```
Comment (FlorianWendelborn) [inline packages/kotti-ui/source/kotti-user-menu/KtUserMenu.vue:82]: Why the ternary here?

**Review by FlorianWendelborn — APPROVED** [review]

**Comment by FlorianWendelborn** [comment]
Can you `autorebase:opt-in`?

File: packages/kotti-ui/source/kotti-user-menu/KtUserMenu.vue (hunk 1 above)
Comment (HansAuger) [reply packages/kotti-ui/source/kotti-user-menu/KtUserMenu.vue]: I removed it now. The reason for this is a VSCode bbug (maybe?) that confused me about the type
Screenshot 2021-11-25 at 14 22 00
Screenshot 2021-11-25 at 14 22 14

**Review by FlorianWendelborn — APPROVED** [review]

File: packages/kotti-ui/source/kotti-user-menu/KtUserMenu.vue (hunk 1 above)
Comment (FlorianWendelborn) [reply packages/kotti-ui/source/kotti-user-menu/KtUserMenu.vue]: Yeah, it’s a visual bug in vetur with `makeProps`. It somehow shows-up as optional even though it’s not. Cause is unknown but doesn’t seem to affect anything besides showing this weirdly

**Pull request merged**